### ISSUE TITLE
Form letter frame

### ISSUE THREAD

**Issue by kennethormandy** [skeskali/MPFormLetters#1]
Hey, this is super useful! I was actually working on something related yesterday, but I was thinking more of the frame for the letter or a phone call script, and less of the content itself. I don’t really have any experience or knowledge on writing letters to MPs, what you’ve done is the more important part. Would you be interested having the site I made auto-fill in some details for your form letters for people? Demo:

[![call-your-mp-1](https://cloud.githubusercontent.com/assets/1581276/22416769/65653c2c-e685-11e6-8668-02395d401d94.gif)](https://call-your-mp.surge.sh)

Basically it’s just a phone call template, but it’ll fill in your MP’s info using your current location. The official postal code search gets the job done, but I thought it was nice to have it integrated into the call script (and then potentially make it one button to send an email version as well).

I have the initial version live at [call-your-mp.surge.sh](https://call-your-mp.surge.sh), and my plan was to open source it anyway. Either my project could pull in your Markdown templates somehow, or I could hard-code them for now, or something else. Let me know if that’s interesting to you at all! Thanks either way.

**Comment by skeskali** [comment]
YES, YES, YES! I was hoping someone would come along and volunteer to do this.

**Comment by kennethormandy** [comment]
Great! I should be able to do this on Friday or Saturday. I’ll try making it something that gets the Markdown from your repo, or I’ll figure out a good way to open a PR to add the site to this repo while keeping the letters at the top level of this repo so they’re still easy to contribute to.

**Comment by kennethormandy** [comment]
Okay! The letter is now getting dynamically fetched from your repository client-side. It’s done in a pretty hard-coded way, but it’s a start. If you were to change some of the content in the letter, it would get updated automatically.

[call-your-mp.surge.sh](https://call-your-mp.surge.sh/)

I also made the postal code lookup thing work, but only as a fallback if you don’t want to use your current location.

[![call-your-mp-2](https://cloud.githubusercontent.com/assets/1581276/22622690/882a39fe-eaf6-11e6-814a-c97a654ca3d5.gif)](https://call-your-mp.surge.sh/)

**Comment by skeskali** [comment]
This is phenomenal - thank you **SO MUCH** for doing this!

**Comment by kennethormandy** [comment]
You’re welcome, thanks for sharing the letters! I’m pretty open to where this could go next if you have any suggestions—making it work with more than just the one letter seems like it would probably make the most sense, but there are a lot of different ways that could work.

The repository [is public now](https://github.com/kennethormandy/call-your-mp) too.
